ontology/ogsf.md

```diff
@@ -18,6 +18,7 @@ tracker: https://github.com/linikujp/OGSF/issues
 activity_status: active
 repository: https://github.com/linikujp/OGSF
 preferredPrefix: OGSF
+domain: phenotype
```

Comment:
@linikujp thoughts?

Reply:
the options are listed at the top of this PR

Reply:
Hi @cthoyt I saw your list of domains, I think it fits investigations.

Reply:
```suggestion
domain: investigations
```
@matentzn what do you think?